src/Hydrators/PropertyHydrator.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+
+namespace ArmadilloEstates\Hydrators;
+
+
+use ArmadilloEstates\Entities\PropertyEntity;
+
+class PropertyHydrator
+{
+    private $database;
+
+   public function __construct(\PDO $database)
+   {
+       $this->database = $database;
+   }
+
+   public function getAllBasicProperties()
+   {
+       $query = $this->database->query("SELECT `Address1`, `Town`, `Type`, `Status`, `id`,  `Image` FROM `properties`;");
```

Comment:
same comments as were in task2b pr

Reply:
yeah this won't be in this PR once Task2b is in Story1